diff --git a/config/paths.js b/config/paths.js
index 718b898..8e06580 100644
--- a/config/paths.js
+++ b/config/paths.js
@@ -52,9 +52,11 @@ module.exports = {
   appBuild: resolveApp('build'),
   appPublic: resolveApp('public'),
   appHtml: resolveApp('public/index.html'),
-  appIndexJs: resolveApp('src/index.js'),
+  appIndexJs: resolveApp('main.js'),
   appPackageJson: resolveApp('package.json'),
   appSrc: resolveApp('src'),
+  appsDir: resolveApp('apps'),
+  configDir: resolveApp('config'),
   yarnLockFile: resolveApp('yarn.lock'),
   testsSetup: resolveApp('src/setupTests.js'),
   appNodeModules: resolveApp('node_modules'),
@@ -72,9 +74,11 @@ module.exports = {
   appBuild: resolveApp('build'),
   appPublic: resolveApp('public'),
   appHtml: resolveApp('public/index.html'),
-  appIndexJs: resolveApp('src/index.js'),
+  appIndexJs: resolveApp('main.js'),
   appPackageJson: resolveApp('package.json'),
   appSrc: resolveApp('src'),
+  appsDir: resolveApp('apps'),
+  configDir: resolveApp('config'),
   yarnLockFile: resolveApp('yarn.lock'),
   testsSetup: resolveApp('src/setupTests.js'),
   appNodeModules: resolveApp('node_modules'),
diff --git a/config/webpack.config.dev.js b/config/webpack.config.dev.js
index 9f3131b..4b475d6 100644
--- a/config/webpack.config.dev.js
+++ b/config/webpack.config.dev.js
@@ -163,7 +163,7 @@ module.exports = {
           // Process JS with Babel.
           {
             test: /\.(js|jsx|mjs)$/,
-            include: paths.appSrc,
+            include: [paths.appSrc, paths.appsDir, paths.configDir],
             loader: require.resolve('babel-loader'),
             options: {
               // @remove-on-eject-begin
